Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.cpp

```diff
@@ -424,6 +424,49 @@ namespace AzToolsFramework
             return newTemplateId;
         }
 
+        void PrefabSystemComponent::UpdateTemplateFilePath(TemplateId templateId, const AZ::IO::PathView& filePath)
+        {
+            auto findTemplateResult = FindTemplate(templateId);
+            if (!findTemplateResult.has_value())
+            {
+                AZ_Warning(
+                    "Prefab", false,
+                    "PrefabSystemComponent::UpdateTemplateFilePath - "
```

Comment:
Nitpick: Specifying the function adds noise to the log. They're not really helpful for artists or designers and these functions can also dump the callstack which will include the same information.

Reply:
yeah I copied this from how it is done in the rest of file but I removed it, since I agree